From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: xNombre <kartapolska@gmail.com>
Date: Fri, 23 Jul 2021 11:37:54 +0200
Subject: [PATCH 16/19] init.rc: Don't enable f2fs iostat by default

It is only intended for performance analysis and can
impact I/O throughput.

Signed-off-by: xNombre <kartapolska@gmail.com>
---
 rootdir/init.rc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 0a75fcb..2551faf 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1213,7 +1213,6 @@ on boot
     write /dev/sys/fs/by-name/userdata/cp_interval 30
     write /dev/sys/fs/by-name/userdata/gc_urgent_sleep_time 50
     write /dev/sys/fs/by-name/userdata/iostat_period_ms 1000
-    write /dev/sys/fs/by-name/userdata/iostat_enable 1
 
     # set readahead multiplier for POSIX_FADV_SEQUENTIAL files
     write /dev/sys/fs/by-name/userdata/seq_file_ra_mul 128
